easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.3-foss-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'PythonPackage'
+
+name = 'RSeQC'
+version = '2.6.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://rseqc.sourceforge.net/'
+description = """RSeQC provides a number of useful modules that can 
+ comprehensively evaluate high throughput sequence data especially RNA-seq 
+ data. Some basic modules quickly inspect sequence quality, nucleotide 
+ composition bias, PCR bias and GC bias, while RNA-seq specific modules 
+ evaluate sequencing saturation, mapped reads distribution, coverage 
+ uniformity, strand specificity, transcript level RNA integrity etc."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [SOURCEFORGE_SOURCE]
+
+dependencies = [
+    ('R', '3.2.3'),
+    ('Python', '2.7.10'),
+]
+
+runtest = "python setup.py test"
+
+sanity_check_paths = {
+    'files': ['bin/bam_stat.py', 'bin/overlay_bigwig.py', 
+              'bin/split_paired_bam.py'],
```

Comment:
anything under 120 characters is fine; if it's below 100 characters, there's absolutely no reason to split imho